### PR TITLE
Shortcuts Panel related refactor

### PR DESCRIPTION
**Pull request by pskowronek**
Shortcuts Panel/Table related refactor to make it a lil' bit easier for #952 implementation.

Mainly:
- code style, tabs -> spaces,
- replaced arrays, indexes with maps, enums etc

### CODE REVIEW

File: mucommander-core/src/main/java/com/mucommander/ui/dialog/pref/general/ShortcutsTable.java [hunk 1]
```diff
@@ -350,7 +376,7 @@ public void removeUpdate(DocumentEvent e) {}
 
         @Override
         public Component getTableCellEditorComponent(JTable table, Object value,
-                boolean isSelected, int row, int col) {
+                                                     boolean isSelected, int row, int col) {
```
Comment (ahadas) [inline mucommander-core/src/main/java/com/mucommander/ui/dialog/pref/general/ShortcutsTable.java:379]: that would also change by the formatter

File: mucommander-core/src/main/java/com/mucommander/ui/dialog/pref/general/ShortcutsTable.java [hunk 2]
```diff
@@ -376,65 +402,69 @@ public void neutralize() {
...
 
-        private KeymapTableModel(ShortcutsTableData data) {
-            super(data.getTableData(), new String[] {Translator.get("shortcuts_table.action_description"),
-                    Translator.get("shortcuts_table.shortcut"),
-                    Translator.get("shortcuts_table.alternate_shortcut")});
-            this.tableData = data;
+        private KeymapTableModel(ShortcutsTableData tableData) {
+            super(new String[] { Translator.get("shortcuts_table.action_description"),
+                                 Translator.get("shortcuts_table.shortcut"),
+                                 Translator.get("shortcuts_table.alternate_shortcut")},
```
Comment (ahadas) [inline mucommander-core/src/main/java/com/mucommander/ui/dialog/pref/general/ShortcutsTable.java:420]: and this

**Review by ahadas — APPROVED** [review]
looks good but this requires a follow up to align the changes with the formatter rules

File: mucommander-core/src/main/java/com/mucommander/ui/dialog/pref/general/ShortcutsTable.java [hunk 3]
```diff
@@ -32,12 +32,8 @@
 import java.awt.event.KeyEvent;
 import java.awt.event.KeyListener;
 import java.awt.image.BufferedImage;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.LinkedList;
-import java.util.List;
+import java.util.*;
```
[the remark this answers is not in the document]
Comment (pskowronek) [reply mucommander-core/src/main/java/com/mucommander/ui/dialog/pref/general/ShortcutsTable.java:35]: ok, I will try [this](https://plugins.jetbrains.com/plugin/6546-adapter-for-eclipse-code-formatter)

**Pull request merged**